helmfile.d/helmfile-09.init.yaml

```diff
@@ -24,12 +24,17 @@ releases:
       pkg: knative
     values:
       - ../values/knative/knative-serving-raw.gotmpl
-  - name: gatekeeper-operator-config
-    installed: {{ $c | get "gatekeeper-operator.enabled" false }}
+  - name: gatekeeper-artifacts
```

Comment:
Artifacts are deployed quite late.
Can we do it ijmmifiatly after deploying gatekeeper?